app/routes/show.js

```diff
@@ -80,8 +80,10 @@ export default Ember.Route.extend(SetPageTitle, {
 	},
 
   setupController: function(controller, model) {
-    var params = this.paramsFor(this.get('routeName'));
-    if (model.show.get('vods.firstObject.chapters.length') || params.seekto) {
+    let params = this.paramsFor(this.get('routeName'));
+    let chapters = model.show.get('vods.firstObject.chapters') || [];
+    chapters = chapters.rejectBy('deleted');
+    if (chapters && params.seekto) {
```

Comment:
In line 83: `let chapters = model.show.get('vods.firstObject.chapters') || [];`

If the vod has no chapters, then `chapters` will be set to `[]`. Won't `chapters` evaluate to true when it's an empty array? If so, should we check `chapters.length` or something?

Reply:
empty arrays are falsy in JS

![image](https://user-images.githubusercontent.com/41680/36160842-32989e00-10b0-11e8-9d37-8fcda9d1e01e.png)

Reply:
actually... I'm smoking rocks... good call